app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    result_dict = {}
+
+    def inner(*args) -> Any:
+        for key, value in result_dict.items():
+            if key == args:
+                print("Getting from cache")
+                return value
```

Comment:
I'm not sure, that is the best solution. I think, that you must check that key exists in dict and return value